news/tasks/extraction.py

```
@@ -0,0 +1,80 @@
+from celery import shared_task
+from typing import List, Dict, Any
+import logging
+
+from news.services.jina_api import JinaReaderService
+from news.models import News
+
+logger = logging.getLogger(__name__)
+
+
+@shared_task(bind=True, max_retries=3)
+def extract_single_article_task(self, article: Dict[str, Any]) -> Dict[str, Any] | None:
```

Ruff (0.14.10)

Unused function argument: `self`

(ARG001)